pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/PulsarAdminBuilderImpl.java

```diff
@@ -47,6 +47,7 @@ public PulsarAdmin build() throws PulsarClientException {
 
     public PulsarAdminBuilderImpl() {
         this.conf = new ClientConfigurationData();
+        this.conf.setConnectionsPerBroker(16);
```

Comment:
Couldn't the default be part of `ClientConfigurationData` constructor?

Reply:
> Couldn't the default be part of `ClientConfigurationData` constructor?

Not really. ClientConfigurationData is designed for PulsarClient, but it's also used in PulsarAdmin client. The current PulsarAdminBuilderImpl is a bit of a hack around ClientConfigurationData.